build-env/docker/docker-debian-11-bullseye/Dockerfile.build

```diff
@@ -0,0 +1,48 @@
+FROM debian:bullseye
```

Comment:
I recommend we pin this floating base image version.  PR against this branch: https://github.com/PelionIoT/distro-pelion-edge/pull/213